### PR TITLE
Build: Add Standard: The Standard Linter/Formatter Gem

### PR DESCRIPTION
**Pull request by joaoGabriel55**
Issue: #21

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -21,14 +34,47 @@ GEM
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.13.0)
     rspec-support (3.13.1)
+    rubocop (1.64.1)
```
Comment (carlospohlodcodeminer42) [inline Gemfile.lock:37]: we need to add it as a development dependency. [example](https://github.com/carlosdanielpohlod/rails_tracepoint_stack/blob/main/rails_tracepoint_stack.gemspec#L28)

File: Gemfile.lock [hunk 2]
```diff
@@ -21,14 +34,47 @@ GEM
...
+      rubocop-ast (>= 1.31.1, < 2.0)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (>= 2.4.0, < 3.0)
+    rubocop-ast (1.31.3)
+      parser (>= 3.3.1.0)
+    rubocop-performance (1.21.1)
+      rubocop (>= 1.48.1, < 2.0)
+      rubocop-ast (>= 1.31.1, < 2.0)
+    ruby-progressbar (1.13.0)
+    standard (1.39.2)
```
Comment (carlospohlodcodeminer42) [inline Gemfile.lock:54]: we need to add it as a development dependency. [example](https://github.com/carlosdanielpohlod/rails_tracepoint_stack/blob/main/rails_tracepoint_stack.gemspec#L28)

**Comment by carlospohlodcodeminer42** [comment]
@joaoGabriel55 can you fix the conflicts please?

**Review by carlosdanielpohlod — APPROVED** [review]
thank you!

**Pull request merged**